testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -3311,14 +3361,22 @@ private void createTestIndex(
     assertThat(successResponse.getSuccess()).isTrue();
   }
 
-  /** @return {@code List} of entries to expect back for given definitions. */
+  /**
+   * Helper method for inserting table entries using so-called "Stringified" values for columns:
+   * this differs a bit from full JSON values and is mostly useful for simple String and number
+   * fields.
+   *
+   * @return {@code List} of entries to expect back for given definitions.
+   */
   private List<Map<String, String>> insertTestTableRows(List<List<String>> rows)
       throws IOException {
     final List<Map<String, String>> insertedRows = new ArrayList<>();
     for (List<String> row : rows) {
       Map<String, String> rowMap = new HashMap<>();
       for (String kv : row) {
-        String[] parts = kv.split(" ");
+        // Split on first space, leave others in (with no limit we'd silently
+        // drop later space-separated parts)
+        String[] parts = kv.split(" ", 2);
```

Comment:
Although no tests exhibited the issue, I had this problem earlier when trying to write a test: "trailing" values were silently just ignored. Instead it will now split String into exactly 2 parts, with the first space.
(previously would have split to however many segments and drop all but 2: something for which there is no use case)